docs/sources/api/_index.md

```diff
@@ -296,6 +303,8 @@ And `<stream value>` is:
 }
 ```
 
+The items in the `values` array are sorted by their timestamp in a descending (when using `direction=backward`) or ascending (when using `direction=forward`) order.
```

Comment:
Suggesting same prose here as in the earlier commit suggestion.
```suggestion
The items in the `values` array are sorted by timestamp.
The most recent item is first when using `direction=backward`.
The oldest item is first when using `direction=forward`.
```

Reply:
done!